pedant/tests.py

```diff
@@ -485,7 +498,8 @@ def render1():
         self.assertFalse(logger2.log.called)
         self.assertEqual(logged_render, original_render)
 
-    @override_settings(TEMPLATE_STRING_IF_INVALID='Fixed String')
+    @skipIf(django.VERSION >= (1, 8), 'Not an issue as of Django 1.8')
```

Comment:
These tests check that Django's cached version of `TEMPLATE_STRING_IF_INVALID` is overwritten when applying the patch decorators. Django no longer caches `TEMPLATE_STRING_IF_INVALID` onto `django.template.base.invalid_var_format_string`, so these tests are no longer relevant. It *does* cache it onto the template Engine, but version of the patcher that I wrote for 1.8 and 1.9 should patch the underlying Engine directly instead of the setting.